core/store/models/common.go

```diff
@@ -662,3 +662,17 @@ func (h ServiceHeader) Validate() (err error) {
 	}
 	return
 }
+
+// CreateEVMTransactionRequest represents a request to create request to submit evm transaction.
+type CreateEVMTransactionRequest struct {
+	IdempotencyKey     string          `json:"idempotencyKey"`
+	ChainID            *utils.Big      `json:"chainID"`
+	ToAddress          *common.Address `json:"toAddress"`
+	FromAddress        common.Address  `json:"fromAddress"`    // optional, if not set we use one of accounts available for specified chain
+	EncodedPayload     string          `json:"encodedPayload"` // hex encoded payload
+	Value              *utils.Big      `json:"value"`
+	ForwarderAddress   common.Address  `json:"forwarderAddress"`
+	FeeLimit           uint32          `json:"feeLimit"`
+	SkipWaitTxAttempt  bool            `json:"skipWaitTxAttempt"`
```

Comment:
I think let us not add these 2 last properties.
These were sort of added for SendEtherRequest, to have sync Tx Send behavior.
For the TXM case here in Config Distribution, we don't want to block our request, for the TXM to actually create an onchain attempt, which could be many seconds or more of wait, and will block the REST call.
Let us just directly return after the Tx is created.